lib/utils/matrix_sdk_extensions/event_extension.dart

```diff
@@ -177,7 +177,6 @@ extension LocalizedBody on Event {
       }.contains(messageType);
 
   bool hideDisplayName(Event? nextEvent) =>
-      isOwnMessage ||
```

Comment:
in mobile, we dont need to show displayname, because it is in different side

Reply:
Updated

![image](https://github.com/user-attachments/assets/f6f3344e-9ebf-4008-8276-3290d9f48316)